gym_solo/envs/solo8_base_env.py

```diff
@@ -102,6 +104,34 @@ def observation_space(self) -> gym.Space:
     """
     return self.obs_factory.get_observation_space()
 
+  def render(self, mode='rgb_array') -> List[List[List[int]]]:
+    """Render the current state of the Solo 8 env.
+    
+    Note that the camera is controlled by the `config` option passed in at
+    construction time.
+
+    Args:
+      mode (str, optional): Rendering mode. Refer to OpenAI Gym documentation. 
+        Defaults to 'rgb_array'.
+
+    Returns:
+      List[List[List[int]]]: a 2D list of RGBA pixel data.
```

Comment:
Isn't a list of list of list a 3D array?

Reply:
it's really a 2d array of RGBA tuple data, so a 2d array of pixel data. Unfortunately, pybullet just represents this as a list instead of a tuple, so I'm setting this type. I kept it as 2d for semantics, but lmk if you thinks its clearer as 3d.

Reply:
I was just confused by three Lists and this being called 2D. Just wanted to verify if this was not a mistake